py/selenium/webdriver/chrome/options.py

```diff
@@ -18,22 +18,21 @@
 import base64
 import os
 import platform
-import warnings
 
 from selenium.webdriver.common.desired_capabilities import DesiredCapabilities
+from ..common.options import ArgOptions
```

Comment:
any reason for the relative import instead of absolute?

Reply:
Habit, I can change it to absolute